index.js

```diff
@@ -0,0 +1,40 @@
+
+/**
+ * Useful api:
+ * top headlines
+ * https://newsapi.org/v2/top-headlines?country=ca&apiKey=<key>
+ * - print total_results,
+ * - loop through articles and print article.title, article.description, article.urlToImage, article.publishedAt, article.url(to be opened in new tab)
+ */
+import news_api_key from './news_api_key.js';
```

Comment:
.htaccess file can be a safe and effective way to keep API keys secure.

Reply:
Thanks Jordan. I will look into it.